### PR TITLE
[OrderByComparator] LPD-26679 Clean up and Apply Singleton in dynamic-data-mapping module part II

### PR DESCRIPTION
**Pull request by MrJohn1911**

### CODE REVIEW

**Comment by MrJohn1911** [comment]
ci:test:forms

**Comment by MrJohn1911** [comment]
ci:report:2571499700

**Comment by MrJohn1911** [comment]
ci:report:2571512499

File: modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/util/comparator/StructureLinkStructureNameComparator.java [hunk 1]
```diff
@@ -24,8 +24,14 @@
...
+		boolean ascending) {
+
+		if (ascending) {
+			return _ASCENDING;
+		}
+
+		return _DESCENDING;
 	}
 
 	public StructureLinkStructureNameComparator(
```
Comment (dantewang) [inline modules/apps/dynamic-data-mapping/dynamic-data-mapping-api/src/main/java/com/liferay/dynamic/data/mapping/util/comparator/StructureLinkStructureNameComparator.java:37]: @MrJohn1911 It seems this constructor has no external usage, either. Please double check.

**Review by dantewang — CHANGES_REQUESTED** [review]

**Comment by dantewang** [comment]
BTW, please change commits like making constant without external usage to private "SF, ...." commits.

**Comment by MrJohn1911** [comment]
ci:test:sf

**Pull request closed without merging**